supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalBackend.scala

```diff
@@ -1,34 +1,56 @@
 package cromwell.backend.impl.local
 
-import cromwell.backend.BackendJobExecutionActor.BackendJobExecutionResponse
+import java.nio.file.FileSystems
+
+import cromwell.backend.BackendJobExecutionActor.{BackendJobExecutionFailedResponse, BackendJobExecutionResponse}
 import cromwell.backend.BackendLifecycleActor.JobAbortResponse
 import cromwell.backend._
+import better.files._
+import org.slf4j.LoggerFactory
 import wdl4s.Call
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
+import scala.language.postfixOps
+import scala.util.Try
 
-class LocalBackend extends BackendJobExecutionActor {
-  /**
-    * Restart or resume a previously-started job.
-    */
-  override def recover(jobDescriptor: BackendJobDescriptor): Future[BackendJobExecutionResponse] = ???
+object LocalBackend {
+  val logger = LoggerFactory.getLogger("LocalBackend")
+}
 
-  /**
-    * Execute a new job.
-    */
-  override def execute(jobDescriptor: BackendJobDescriptor): Future[BackendJobExecutionResponse] = ???
+class LocalBackend(override val workflowDescriptor: BackendWorkflowDescriptor,
+                   override val configurationDescriptor: BackendConfigurationDescriptor,
+                   override val calls: Seq[Call]) extends BackendJobExecutionActor {
+
+  override protected implicit def ec: ExecutionContext = scala.concurrent.ExecutionContext.global
```

Comment:
This feels icky. Doesn't it already acquire the actor system context by default?